aea/contracts/base.py

```diff
@@ -121,3 +121,62 @@ def from_config(cls, configuration: ContractConfig) -> "Contract":
         #     contract_interface = json.load(interface_file)
 
         return contract_class(configuration)
+
+    # TODO if 'ContractApiMessage' was a default package
```

Comment:
Ok; makes sense to have it as a default package in that case

Reply:
I think it should be included by default, like the default protocol and the error skill.

Reply:
the reason is, the ledger connection is the only reasonable way the framework helps the agent to perform its economic activity, which is the essence of an AEA.

Reply:
Actually; we should not expose the message here at all. That conflates concepts. So this is mute.